maven/tests/com.liferay.ide.maven.core.tests/src/com/liferay/ide/maven/core/tests/SpringPortletProjectTests.java

```diff
@@ -39,6 +40,7 @@ private NewLiferayPluginProjectOp newMavenProjectOp( String name, String framewo
     }
 
     @Test
+    @Ignore
```

Comment:
Dont add these, these were just for your own local testing, shouldn't be in this commint.